index.html

```diff
@@ -128,8 +124,9 @@ <h2 class="big">Event <span class="highlight">Program</span></h2>
       <div class="entry entry-with-icon">
         <div class="time"></div>
         <div class="details">
-          <div class="entry-name speaker">You?</div>
-          <div class="title"><a href="mailto:contact@rubynl.org?subject=I'd like to give a lightning talk">Tell us</a> about your talk!</div>
+          <div class="entry-name speaker">Amanda Perino</div>
```

Comment:
This looks like so:
![image](https://user-images.githubusercontent.com/625294/223980835-471d5909-9de0-4793-8d25-d5ee8b8214b9.png)